### PR TITLE
Manifestv2: install

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    [resources.sources]
+    [resources.sources.main_16_oca]
+    url = "https://github.com/OCA/OCB/archive/9d5ff52da5ec65a21a00a46e21783a2c6188dd80.tar.gz"
+    sha256 = "3c3a95ec627e3807c85b0b2fd13a484b0b990426efb4905681aaee8768386e39"
+    prefetch = false
+
+    [resources.sources.main_16]
+    url = "https://github.com/odoo/odoo/archive/2a88076c1d1df4920bfd139c94862a894d8ef3bc.tar.gz"
+    sha256 = "39d23f01d9a095d75460829acfd53b74bcfc273a074d6eae163eedcaeeeb58c7"
+    prefetch = false
```
Comment (alexAubin) [inline manifest.toml:85]: Randomly passing but it sounds like a big meh to be able to choose between the twos ... The OCA/OCB one should be equivalent but with more bugfixes etc. I would just install OCA/OCB and voila...

File: manifest.toml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+    [resources.sources.main_9]
+    url = "https://github.com/odoo/odoo/archive/7132a15c38a3b5e5ec232404d21aacfa2ac5874e.tar.gz"
+    sha256 = "f4c522da29201835435b01c651032d07c2d0c0e8a14b30f9a784cb531c5ba250"
+    prefetch = false
+
+    [resources.sources.main_8]
+    url = "https://github.com/odoo/odoo/archive/9e8f70e4849b0eeaca8b5cf51372ecfa23dc561b.tar.gz"
+    sha256 = "51d5db432364d272efe8c7b8f93757a63d762393a9163307f52a2f4f950ec4e6"
+    prefetch = false
```
Comment (alexAubin) [inline manifest.toml:135]: It feels like heresy to allow anybody to install a version lower than 12 (and even 12 is uugh so old)

File: manifest.toml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+    [resources.sources.main_15_oca]
+    url = "https://github.com/OCA/OCB/archive/1ab48bd4c89f21691b34f349338405fcbb85dea4.tar.gz"
+    sha256 = "23b8902b6fc22e4ce36f6dcc7178257235663e9911b61b70b70aeca0fdb36331"
+    prefetch = false
+
+    [resources.sources.main_15]
+    url = "https://github.com/odoo/odoo/archive/1ab48bd4c89f21691b34f349338405fcbb85dea4.tar.gz"
+    sha256 = "93e12c7574e73403d4da456cf612fd5542d961e302fdf9c3a87cdeca9146184d"
+    prefetch = false
```
Comment (alexAubin) [inline manifest.toml:95]: I would discourage proposing version 15 to be installed, OCA provides many addons, but most addons are only available for even-numbered version I think ...

File: manifest.toml (hunk 1 above)
Comment (Salamandar) [reply manifest.toml:85]: Fine with me, i don't understand what OCA builds are :D

Comment (Salamandar) [reply manifest.toml]: Done.

File: manifest.toml (hunk 2 above)
Comment (Salamandar) [reply manifest.toml]: Done.

File: manifest.toml (hunk 3 above)
Comment (Salamandar) [reply manifest.toml]: Done.

File: manifest.toml [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+    [install.app_version]
+    ask.en = "Choose the version you want"
+    ask.fr = "Choisissez la version désirée"
+    type = "string"
+    choices = ["11", "12", "14", "15", "16"]
```
Comment (alexAubin) [inline manifest.toml:45]:
```suggestion
    choices = ["12", "14", "16"]
```

**Pull request merged**